app/models/service_template_ansible_playbook.rb

```diff
@@ -52,6 +52,15 @@ def self.create_catalog_item(options, auth_user)
     end
   end
 
+  def self.manager_valid?(manager)
```

Comment:
can this work?
```
manager.configuration_script_sources.count.nonzero? &&
manager.configuration_script_payloads.count.nonzero? &&
manager.credentials.where(:type => ManageIQ::Providers::AnsibleTower::AutomationManager::MachineCredential).count.nonzero?
```
note, `ManageIQ::Providers::AnsibleTower::AutomationManager::MachineCredential).count.nonzero?` might now be `ManageIQ::Providers::EmbedAnsible::AutomationManager::MachineCredential).count.nonzero?`